### PR TITLE
fix: sdk links

### PR DESCRIPTION
**Pull request by royscheepens**

### CODE REVIEW

File: pages/service/index.mdx [hunk 1]
```diff
@@ -22,11 +22,11 @@ Obtain your API key. If you haven't received any API key yet, please contact bui
 
 ### Step 4 - Pick an implementation strategy
 
-Pick an implementation strategy. You can either connect to the HTTP API directly, or use one of our provided SDKs. We currently provide a [Node.js (Typescript)](https://github.com/Merit-Circle/beam-node), and [C#](https://github.com/Merit-Circle/beam-csharp) SDK, but are happy to provide you with a client based on whatever language you need to support
+Pick an implementation strategy. You can either connect to the HTTP API directly, or use [one of our provided SDKs](https://github.com/Merit-Circle/beam-sdk). We currently provide a [Node.js (Typescript)](https://www.npmjs.com/package/@onbeam/node) and C# SDK, but are happy to provide you with a client based on whatever language you need to support
```
Comment (lewinskimaciej) [inline pages/service/index.mdx:25]: https://www.nuget.org/packages/Beam

Comment (royscheepens) [reply pages/service/index.mdx:25]: Just what I was looking for!

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Beam is currently in a developer preview, and both the service and the documenta
 
 ### TL;DR
 
-If you would like to get your hands dirty without reading our documentation, we strongly advise you to get started with one of our implementations ready [SDKs](https://docs.onbeam.com/service/sdk). We currently provide SDKs for Node.js and C#, but are happy to facilitate more. If you are interested in an SDK for your implementation language which we don't provide yet, please open an [issue](https://github.com/Merit-Circle/beam-docs/issues/new).
+If you would like to get your hands dirty without reading our documentation, we strongly advise you to get started with one of our implementations ready [SDKs](https://docs.onbeam.com/service/sdk). We currently provide SDKs for Node.js and C#, but are happy to facilitate more. If you are interested in an SDK for your implementation language which we don't provide yet, please open an [issue](https://github.com/Merit-Circle/beam-sdk/issues/new).
```
Comment (royscheepens) [inline README.md:11]: @lewinskimaciej  @RobbyUitbeijerse changed the link at the end to open an issue on the SDK repo instead of the Docs repo.

**Comment by royscheepens** [comment]
Should not have branched off develop
